test/common/protobuf/utility_test.cc

```diff
@@ -1001,6 +1001,20 @@ type_url: type.googleapis.com/envoy.unknown.Message
   EXPECT_TRUE(TestUtility::protoEqual(expected, actual));
 }
 
+TEST_F(ProtobufUtilityTest, RedactEmptyTypeUrlTypedStruct) {
+  udpa::type::v1::TypedStruct actual;
+  MessageUtil::redact(actual);
+  udpa::type::v1::TypedStruct expected = actual;
```

Comment:
Here and the test below: `expected` gets a copy of `actual` after it's been (potentially) modified by the call to `MessageUtil::redact`. The equality comparison on the next line will always return true in this case. You probably want to reverse the order of lines 1006 and 1007? I think the return value of  `MessageUtil::redact` should be verified too.

Reply:
+1

Reply:
ah sorry. I overlooked.